neon_intrinsics/advsimd.rst

```diff
@@ -141,12 +141,16 @@ Document history
 +------------+-------------------------+-------------------------+
 |F           |30 May 2020              |Version ACLE Q2 2020     |
 +------------+-------------------------+-------------------------+
-|G           |30 October 2020          |Version ACLE Q2 2020     |
+|G           |30 October 2020          |Version ACLE Q3 2020     |
 +------------+-------------------------+-------------------------+
-|H           | |date-of-issue|         | |release|               |
+|H           |02 July 2021             |              2021Q2     |
 +------------+-------------------------+-------------------------+
 
 
+Changes between next release  and 2021Q2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Fixed the guard macro for the base intrinsics.
```

Comment:
please add empty lines between the `~~~~~~` and the item (also in the template).

Reply:
Just to make sure: I should _not_ follow the style in https://raw.githubusercontent.com/ARM-software/acle/main/main/acle.rst, then?

Reply:
I am the victim of my own nitpicking. Please do follow the style you refer to. I will resolve this comment... no action on your side.

Reply:
Not needed, sorry.